habitat-lab/habitat/core/utils.py

```diff
@@ -152,12 +155,23 @@ def iterencode(self, o, _one_shot=False):
         default_repr = lambda x: format(x, ".5f")
 
         def floatstr(
```

Comment:
More of a personal note.  The docstring for the nested function will not display on the HTML.

Reply:
That's fine. Good to note.